src/games/strategy/triplea/settings/SettingInputComponentFactory.java

```diff
@@ -134,14 +136,14 @@ private static JPanel createRadioButtonPanel(
 
   }
 
-  private static <Type extends HasDefaults> SettingInputComponent build(
+  private static <T extends HasDefaults> SettingInputComponent<T> build(
```

Comment:
Favor spelling out  "T"
In this case 'Type' was weak to be begin with, should have been named 'SettingsType'
Spelling out the 'T' is a good thing in the long run, particularly helpful in not essential when dealing with multiple generic types.